.gitignore

```diff
@@ -41,3 +41,5 @@ cache.json
 docker-compose.yml
 utils.zsh
 test.db
+.idea/discord.xml
+.idea/dictionaries/
```

Comment:
https://stackoverflow.com/questions/28147162/how-to-create-a-shared-editable-dictionary-for-intellij-idea-spellchecking

Reply:
this is still outstanding